homework-1/cash_register.py

```diff
@@ -0,0 +1,73 @@
+
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+Your class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+
+class CashRegister:
+
+    def __init__(self, total_items):
+
+        self.total_items = dict(total_items)  # {'item': 'price'}
+        self.total_price = 0
+        self.discount = .5
+
+    def add_item(self, item, price):
+        self.total_items.update({item: price})
+        print("Item added:", item)
+
+
+
+    def remove_item(self, item):
+        self.total_items.pop(item)
+        print("Item removed:", item)
```

Comment:
Not bad, however, if a user tries to remove an item that does not exist...this would throw a key error. Not compulsory in this case, but in real-world practice, it is really necessary.